docs/indepth/installation.rst

```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
 
```

Comment:
Not sure why we even bother enabling the root account. All the subsequent steps could use `sudo`, and some of them do